### PR TITLE
ci: k8s: crio: Follow up patches to have CRI-O also working as part of our CI

### PR DESCRIPTION
**Pull request by fidencio**
This is the last set of patches needed to get the CRI-O CI working as part of GHA.

### CODE REVIEW

File: tests/integration/kubernetes/tests_common.sh [hunk 1]
```diff
@@ -46,7 +46,7 @@ exec_host() {
 	command+='; echo -en \\n$?'
 	output="$(kubectl debug -qit "${node}" --image=alpine:latest -- chroot /host bash -c "${command}")"
 	kubectl get pods -o name | grep node-debugger | xargs kubectl delete > /dev/null
-	exit_code="$(echo "${output}" | tail -1)"
-	echo "$(echo "${output}" | head -n -1)"
+	exit_code="$(echo "${output}" | tail -1 | tr -d '\r')"
```
Comment (gkurz) [inline tests/integration/kubernetes/tests_common.sh:49]: What about adding some comment to clearly indicate this is an acceptable workaround until we know exactly what's happening here ?

File: tests/integration/kubernetes/tests_common.sh [hunk 2]
```diff
@@ -46,7 +46,7 @@ exec_host() {
 	command+='; echo -en \\n$?'
 	output="$(kubectl debug -qit "${node}" --image=alpine:latest -- chroot /host bash -c "${command}")"
```
Comment (gkurz) [inline tests/integration/kubernetes/tests_common.sh:47]: FWIW I had this change in my local repo:

```
-       output="$(kubectl debug -qit "${node}" --image=alpine:latest -- chroot /host bash -c "${command}")"
+       output="$(kubectl debug -qit "${node}" --image=alpine:latest -- chroot /host bash -c "${command}" | tr -d '\r')"
```

**Comment by fidencio** [comment]
Adding the `force-skip-ci` to save a few bucks on the jenkins tests, which are not related to this PR at all.

**Review by gkurz — APPROVED** [review]
Yeah ! Thanks @fidencio !

**Review by littlejawa — APPROVED** [review]
lgtm, thanks @fidencio  !

**Pull request merged**